applications/mlflow/charts/mlflow/templates/crd-check-clusterrole.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.mlflow.crdCheck.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "mlflow.fullname" . }}-crd-check-role
```

Comment:
Just a nit, fullname handles truncating to 64 characters, adding -crd-check-role after it could pretty quickly fail. This only is giong to matter if fullname is exposed to user input.

same below.

Reply:
yeah, good point. should probably do another helper function for the name on this one, and make sure its being truncated